apps/prairielearn/src/lib/ai.ts

```diff
@@ -47,8 +47,10 @@ export function calculateResponseCost({
 
   const modelPricing = config.costPerMillionTokens[model];
 
-  const cachedInputTokens = usage.cachedInputTokens ?? 0;
-  const inputTokens = (usage.inputTokens ?? 0) - cachedInputTokens;
+  // TODO: this doesn't take into account cache writes, which some providers
+  // (e.g. Anthropic) charge for.
```

Comment:
I'm deliberately doing nothing about this for now. If anyone feels differently, let me know.

Reply:
I think this is fine as long as cache writes are somewhat-cheap compared to the rest.